# Partículas — boletín II

### Observables: sección eficaz, luminosidade, vida media, invariante $s$, fracción e anchura

Jose A. Hernando

*Departamento de Física de Partículas. Universidade de Santiago de Compostela*

Curso 2026/27

In [1]:
# Configuración e importacións
%matplotlib inline

import numpy as np
from scipy.stats import expon

# Constantes e masas (GeV, s, cm)
hbar  = 6.582e-25      # GeV s
c     = 3.0e10         # cm/s
N_A   = 6.022e23       # 1/mol
m_p   = 0.93827        # GeV
m_pi0 = 0.13498        # GeV
m_tau = 1.7769         # GeV
m_Z   = 91.19          # GeV
nb, pb = 1e-33, 1e-36  # cm^2

## Obxectivos

Os exercicios seguen a orde da sección *Observables* do capítulo de introdución:

* **Sección eficaz e luminosidade** (1): a luminosidade dun branco fixo.
* **Vida media** (2): cantos pares se producen e canto voan.
* **O invariante $s$** (3–8): masa invariante, un pico sobre o fondo (cunha gráfica e con datos
  reais), colisionador fronte a branco fixo e enerxías limiar.
* **Fracción de desintegración** (9) e **anchura** (10).

**Notación.** Unidades naturais, $\hbar = c = 1$, salvo que se diga o contrario. $e$, $\mu$, $\tau$ son os
leptóns con carga negativa e $e^+$, $\mu^+$, $\tau^+$ as súas antipartículas; o signo $+$ separa todas as
partículas. LAB é o sistema do laboratorio e CM o do centro de masas; salvo que se diga o contrario,
os datos están no LAB.

## Sección eficaz e luminosidade

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 1 — Luminosidade dun branco fixo</b></p>

Calcula a luminosidade equivalente dun feixe de protóns de intensidade $10^{13}$ s$^{-1}$ que incide sobre
un tanque de 10 cm de lonxitude de hidróxeno líquido, de densidade $\rho = 60$ kg/m$^3$. Compáraa coa do
LHC, $10^{34}$ cm$^{-2}$s$^{-1}$.

</div>

In [2]:
I    = 1e13                      # protóns/s
rho  = 0.060                     # g/cm^3
l    = 10                        # cm
n    = rho * N_A / 1.008         # protóns (núcleos de H) por cm^3
lum  = I * n * l
print('n = {:.1e} cm^-3,  L = {:.1e} cm^-2 s^-1,  L / L_LHC = {:.0f}'.format(n, lum, lum / 1e34))

n = 3.6e+22 cm^-3,  L = 3.6e+36 cm^-2 s^-1,  L / L_LHC = 358


<details>
<summary><b>Solución</b></summary>

En branco fixo, $\nu = \phi\,\sigma\,N_b = (I/S)\,\sigma\,(n S \ell) = \sigma\,I\,n\,\ell$; comparando con
$\nu = \mathcal{L}\sigma$, $\mathcal{L} = I\,n\,\ell$ (non depende da sección do feixe).
$n = \rho N_A/A = 3.6\times10^{22}$ cm$^{-3}$, logo $\mathcal{L} = 3.6\times10^{36}$ cm$^{-2}$s$^{-1}$,
unhas 400 veces a do LHC. (Aproximación de branco delgado: vale se $\ell \ll \lambda$; aquí
$\lambda \sim 10$ m para $\sigma \sim 30$ mb.)

</details>

## Vida media

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 2 — Pares $\tau + \tau^+$ no pico do $Z$</b></p>

A sección eficaz $e + e^+ \to \tau + \tau^+$ é 1.7 nb a $\sqrt s = 91$ GeV.

1. Se a luminosidade é $10^{32}$ cm$^{-2}$s$^{-1}$, cal é a taxa de produción de pares $\tau + \tau^+$?
2. Calcula a enerxía, o momento e a distancia media percorrida polos $\tau$ ($\tau_\tau = 0.29$ ps).
3. É posible que un $\tau$ concreto percorra 4 mm? Con que probabilidade?
4. No plano transversal, que relación gardan os momentos do $\tau$ e do $\tau^+$?

</div>

In [3]:
sigma, lum = 1.7 * nb, 1e32
E   = 91 / 2
p   = np.sqrt(E**2 - m_tau**2)
d   = p / m_tau * c * 0.29e-12           # gamma beta c tau, en cm
print('nu = {:.2f} /s'.format(sigma * lum))
print('E = {:.1f} GeV, p = {:.2f} GeV, gamma beta = {:.1f}, <d> = {:.2f} mm'.format(E, p, p / m_tau, 10 * d))
print('P(d > 4 mm) = {:.2f}'.format(expon.sf(0.4, scale=d)))

nu = 0.17 /s
E = 45.5 GeV, p = 45.47 GeV, gamma beta = 25.6, <d> = 2.23 mm
P(d > 4 mm) = 0.17


<details>
<summary><b>Solución</b></summary>

1. $\nu = \sigma\mathcal{L} = 1.7\times10^{-33}\times10^{32} = 0.17$ s$^{-1}$.
2. $E = \sqrt s/2 = 45.5$ GeV, $p = 45.47$ GeV, $\gamma\beta = p/m = 25.6$;
   $\langle d\rangle = \gamma\beta c\tau = 25.6\times87\ \mu$m $= 2.2$ mm.
3. Si: a distancia segue unha exponencial de media 2.2 mm; $P(d > 4\ \mathrm{mm}) = e^{-4/2.2} = 0.16$.
4. Opostos e iguais en módulo (o CM coincide co laboratorio).

</details>

## O invariante $s$

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 3 — Masa invariante de tres protóns</b></p>

Tres protóns de $p = 3$ GeV emítense nun plano formando $120^\circ$ entre si. Cal é a súa masa invariante?

</div>

<details>
<summary><b>Solución</b></summary>

Os tres momentos suman cero, así que $M = \sum E_i = 3\sqrt{9 + 0.938^2} = 9.43$ GeV.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 4 — Un pico sobre o fondo</b></p>

A figura mostra os sucesos con **catro leptóns** (dous pares $e + e^+$ ou $\mu + \mu^+$) que rexistrou
ATLAS en colisións $p + p$ a $\sqrt s = 13.6$ TeV (29 fb$^{-1}$), en función da masa invariante dos
catro, $m_{4l}$. Os puntos son os datos; os histogramas, o que se espera dos procesos coñecidos que
tamén dan catro leptóns (o *fondo*).

| |
| :--: |
| <img src="../imgs/higgs_4l_sin_senal.png" width = 420 align = "center"> |
| $m_{4l}$ en ATLAS, co fondo esperado [ATLAS, $\sqrt s = 13.6$ TeV; figura retocada]. |

1. Hai algo nos datos que o fondo non explique? Onde? Estima cantos sucesos de máis hai e a masa da
   partícula que os produce.
2. Nun deses sucesos mídense, no LAB, os momentos (en GeV)
   $e: (54, 0, 0)$, $e^+: (-38, 0, 0)$, $\mu: (0, 0, 19)$, $\mu^+: (-8, 15, 0)$.
   Desprecia a masa dos leptóns. Calcula a masa invariante dos catro. É compatible co pico?
   Por que se usa a masa invariante e non a enerxía total dos catro leptóns?
3. Calcula a masa invariante do par $e + e^+$ e a do par $\mu + \mu^+$. Que partícula se desintegra en
   cada par? Que partícula é a nai?
4. Por que un dos dous bosóns intermedios non pode ter a súa masa?

</div>

In [4]:
def masa(*ps):
    # masa invariante de partículas sen masa, dados os seus trimomentos
    E = sum(np.linalg.norm(p) for p in ps)
    P = np.sum(ps, axis=0)
    return np.sqrt(E**2 - P @ P)

e, ep, mu, mup = map(np.array, [(54, 0, 0), (-38, 0, 0), (0, 0, 19), (-8, 15, 0)])
print('m4l = {:.1f} GeV,  m_ee = {:.1f} GeV,  m_mumu = {:.1f} GeV'.format(
      masa(e, ep, mu, mup), masa(e, ep), masa(mu, mup)))

m4l = 125.4 GeV,  m_ee = 90.6 GeV,  m_mumu = 25.4 GeV


<details>
<summary><b>Solución</b></summary>

| |
| :--: |
| <img src="../imgs/higgs_4l_atlas.png" width = 420 align = "center"> |
| A figura orixinal, co sinal do Higgs (azul) e a incerteza do fondo (raiado). |

1. Entre 120 e 130 GeV hai $8 + 11 + 10 + 10 + 7 = 46$ sucesos onde o fondo espera $\simeq 17$: uns
   **30 sucesos de máis**, agrupados nun pico en $m_{4l} \simeq 125$ GeV. Fóra do pico os datos seguen o
   fondo (con flutuacións de $\pm\sqrt N$). Unha partícula de 125 GeV que se desintegra en catro leptóns:
   o **bosón de Higgs**. A anchura do pico, duns poucos GeV, é a resolución do detector; a do Higgs
   é de 4 MeV.
2. $E = 54 + 38 + 19 + 17 = 128$ GeV e $\vec P = (8, 15, 19)$ GeV, $|\vec P|^2 = 650$ GeV$^2$:
   $m_{4l} = \sqrt{16384 - 650} = 125.4$ GeV, no pico. A enerxía total depende do sistema de referencia
   (aquí o sistema dos catro leptóns móvese con $\beta = 25.5/128 = 0.20$), e dun suceso a outro
   cambia co movemento do Higgs; a masa invariante é a mesma en todos os sistemas e é a masa da
   nai.
3. $m_{ee} = \sqrt{92^2 - 16^2} = 90.6$ GeV: un $Z$. $m_{\mu\mu} = \sqrt{36^2 - 650} = 25.4$ GeV: outro $Z$,
   pero lonxe da súa masa. A cadea é $H \to Z + Z^* \to e + e^+ + \mu + \mu^+$.
4. Dous $Z$ coa súa masa necesitan $2m_Z = 182$ GeV $> m_H = 125$ GeV: non hai enerxía. Un sae coa súa masa e
   o outro, **virtual** ($Z^*$), coa que sobra: $m_{Z^*} \le m_H - m_Z \simeq 34$ GeV. Por iso o segundo
   par ten sempre unha masa invariante pequena.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 5 — O Higgs nos datos abertos de CMS (co ordenador)</b></p>

Refai a figura do exercicio anterior con datos reais: os 278 sucesos con catro leptóns de CMS de 2011 e
2012 ($\sqrt s = 7$ e 8 TeV, 14 fb$^{-1}$), publicados no CERN Open Data Portal. O ficheiro
`datos/higgs_4l_cms.csv` dá os cuadrimomentos dos catro leptóns de cada suceso, e
`datos/higgs_4l_cms_esperados.csv`, os sucesos esperados de cada fondo e dun Higgs de 125 GeV, en
intervalos de 3 GeV de $m_{4l}$.

1. Calcula $m_{4l}$ de cada suceso a partir dos cuadrimomentos.
2. Emparella os leptóns en dous pares de carga oposta e mesmo sabor; chama $Z_1$ ao máis próximo a $m_Z$.
   Fai o histograma de $m_{Z_1}$ e $m_{Z_2}$ para os sucesos con $121 < m_{4l} < 130$ GeV. Que relación hai entre
   $m_{Z_1}$, $m_{Z_2}$ e $m_{4l}$?
3. Debuxa o espectro de $m_{4l}$ entre 70 e 181 GeV sobre os fondos empilados. De que é o pico en 91 GeV?
   Onde hai un exceso?
4. Conta os sucesos observados e o fondo esperado en $121 < m_{4l} < 130$ GeV. Que probabilidade hai de
   que o fondo só dea tantos ou máis? Calcúlaa coa distribución de Poisson e simulando experimentos.
   É un descubrimento?

**Este exercicio faise no seu propio caderno guiado:
[boletin_II_2627_higgs_4l.ipynb](boletin_II_2627_higgs_4l.ipynb)**, cunha tarefa por apartado e as súas solucións.

</div>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 6 — O LHC en branco fixo</b></p>

No LHC colisionan protóns con $\sqrt s = 14$ TeV. Que enerxía ten cada protón? Se a colisión fose en
branco fixo, que enerxía necesitaría o protón incidente?

</div>

In [5]:
s = 14e3**2                               # GeV^2
E_fijo = (s - 2 * m_p**2) / (2 * m_p)
print('colisionador: E = {:.0f} GeV;  branco fixo: E = {:.2e} GeV = {:.1e} eV'.format(
      np.sqrt(s) / 2, E_fijo, E_fijo * 1e9))

colisionador: E = 7000 GeV;  branco fixo: E = 1.04e+08 GeV = 1.0e+17 eV


<details>
<summary><b>Solución</b></summary>

No colisionador (CM), $s = (2E)^2$: $E = 7$ TeV. En branco fixo, $s = (E + m_p)^2 - p^2 = 2m_p^2 + 2m_pE$:

$$
E = \frac{s - 2m_p^2}{2m_p} \simeq \frac{(14\times10^{3})^2}{2\times0.938}\ \mathrm{GeV} = 1.04\times10^{8}\ \mathrm{GeV}
= 1.04\times10^{5}\ \mathrm{TeV} \simeq 10^{17}\ \mathrm{eV},
$$

enerxía que só alcanzan os raios cósmicos.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 7 — Produción de antiprotóns</b></p>

Para producir antiprotóns faise incidir un feixe de protóns sobre un branco de hidróxeno mediante
$p + p \to p + p + p + \bar p$.

1. Comproba que a reacción está permitida.
2. Determina a enerxía mínima no CM.
3. Determina a enerxía e o momento limiar do protón incidente no laboratorio.

</div>

In [6]:
s_min = (4 * m_p)**2
E = (s_min - 2 * m_p**2) / (2 * m_p)
print('sqrt(s)_min = {:.2f} GeV;  E = {:.2f} GeV, p = {:.2f} GeV, T = {:.2f} GeV'.format(
      np.sqrt(s_min), E, np.sqrt(E**2 - m_p**2), E - m_p))

sqrt(s)_min = 3.75 GeV;  E = 6.57 GeV, p = 6.50 GeV, T = 5.63 GeV


<details>
<summary><b>Solución</b></summary>

1. Conserva a carga ($2 \to 2$) e o número bariónico ($2 \to 3 - 1 = 2$): o $\bar p$ créase xunto a un protón.
2. $\sqrt{s}_{\min} = 4m_p = 3.75$ GeV.
3. $2m_p^2 + 2m_pE = 16m_p^2 \Rightarrow E = 7m_p = 6.57$ GeV; $p = \sqrt{48}\,m_p = 6.50$ GeV; $T = 6m_p = 5.63$ GeV.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 8 — Fotoprodución de pións e o corte GZK</b></p>

Considera $\gamma + p \to p + \pi^0$.

1. Cal é a enerxía mínima dos fotóns se inciden sobre protóns en repouso?
2. Que enerxía mínima deben ter os protóns ultrarrelativistas dos raios cósmicos para que a reacción
   teña lugar cos fotóns de 1 meV do fondo cósmico de microondas?
3. Xusto por riba do limiar a sección eficaz é $\sigma = 0.6$ mb e a densidade de fotóns do fondo é
   $\rho_\gamma = 4.1\times10^{8}$ m$^{-3}$. Calcula a lonxitude de interacción do protón. É grande ou
   pequena a escala cosmolóxica? (1 Mpc $= 3.09\times10^{22}$ m.)

</div>

In [7]:
s_umbral = (m_p + m_pi0)**2
E_gamma  = (s_umbral - m_p**2) / (2 * m_p)
E_p      = s_umbral / (4 * 1e-12)                 # fotón de 1 meV = 1e-12 GeV, choque frontal
lam      = 1 / (4.1e8 * 0.6e-31)                  # m  (0.6 mb = 0.6e-31 m^2)
print('E_gamma = {:.0f} MeV;  E_p = {:.1e} eV;  lambda = {:.1e} m = {:.1f} Mpc'.format(
      1e3 * E_gamma, 1e9 * E_p, lam, lam / 3.09e22))

E_gamma = 145 MeV;  E_p = 2.9e+20 eV;  lambda = 4.1e+22 m = 1.3 Mpc


<details>
<summary><b>Solución</b></summary>

No limiar, $s = (m_p + m_\pi)^2$.

1. $s = (E_\gamma + m_p)^2 - E_\gamma^2 = m_p^2 + 2m_pE_\gamma \Rightarrow
   E_\gamma = \dfrac{(m_p + m_\pi)^2 - m_p^2}{2m_p} = 145$ MeV.
2. Con $E_p \simeq p_p$: $s \simeq 2E_\gamma E_p(1 - \cos\theta)$, mínimo en choque frontal ($\cos\theta = -1$):
   $s = 4E_\gamma E_p \Rightarrow E_p = \dfrac{(m_p + m_\pi)^2}{4E_\gamma} = 2.9\times10^{20}$ eV.
   Por riba, os protóns perden enerxía co fondo cósmico (corte GZK).
3. $\lambda = 1/(\rho_\gamma\sigma) = 4\times10^{22}$ m $\simeq 1.3$ Mpc, fronte aos $\sim10^{4}$ Mpc do
   universo observable: **minúscula**. Un protón de $10^{20}$ eV non pode vir de lonxe.
   (Matiz honesto: 1.3 Mpc é a lonxitude de interacción *moi por riba* do limiar. A distancia que de
   verdade importa é a de perda de enerxía, máis longa —da orde de 100 Mpc—, porque preto do limiar só
   participa a cola enerxética do espectro de corpo negro e porque cada colisión se leva só un
   $\sim20\,\%$ da enerxía do protón. A conclusión non cambia: hai un horizonte GZK, e os raios cósmicos
   de enerxía extrema teñen que proceder do universo próximo.)

</details>

## Fracción de desintegración e anchura

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 9 — Higgs a dous fotóns no LHC</b></p>

A sección eficaz de produción do Higgs é 50 pb en colisións $p + p$ a $\sqrt s = 8$ TeV. Se
$\mathcal{B}(H \to \gamma + \gamma) = 2.2\times10^{-3}$, cantos sucesos deste tipo se producen nun ano
($10^7$ s) con $\mathcal{L} = 10^{34}$ cm$^{-2}$s$^{-1}$? Que luminosidade integrada fai falta para
producir 100?

</div>

In [8]:
sigma, Br, lum, T = 50 * pb, 2.2e-3, 1e34, 1e7
print('N = {:.1e};  L_int para 100 = {:.0f} pb^-1'.format(sigma * Br * lum * T, 100 / (50 * Br)))

N = 1.1e+04;  L_int para 100 = 909 pb^-1


<details>
<summary><b>Solución</b></summary>

$N = \sigma\,\mathcal{B}\,\mathcal{L}\,T = 50\times10^{-36}\times2.2\times10^{-3}\times10^{34}\times10^{7} = 1.1\times10^{4}$.
$\mathcal{L}_{\rm int} = N/(\sigma\mathcal{B}) = 100/(50\ \mathrm{pb}\times2.2\times10^{-3}) = 909$ pb$^{-1}$
$\simeq 0.9$ fb$^{-1}$.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Exercicio 10 — Anchura e vida media</b></p>

A vida media do $\pi^0$ é $\tau = 8.4\times10^{-17}$ s. Cal é a súa anchura de desintegración? A anchura
do mesón $\eta$ é $\Gamma_\eta = 1.3$ keV. Cal é a súa vida media?

</div>

In [9]:
print('Gamma(pi0) = {:.1f} eV;  tau(eta) = {:.1e} s'.format(1e9 * hbar / 8.4e-17, hbar / 1.3e-6))

Gamma(pi0) = 7.8 eV;  tau(eta) = 5.1e-19 s


<details>
<summary><b>Solución</b></summary>

$\Gamma = \hbar/\tau$: $\Gamma_{\pi^0} = 6.58\times10^{-16}$ eV s $/ 8.4\times10^{-17}$ s $= 7.8$ eV;
$\tau_\eta = \hbar/\Gamma_\eta = 6.58\times10^{-16}/1.3\times10^{3} = 5.1\times10^{-19}$ s.
Anchuras de eV–keV para desintegracións electromagnéticas; o $\eta$ vive menos porque ten máis canles
abertas.

</details>

## Datos

**Constantes.** $\hbar c = 197.3$ MeV fm; $\hbar = 6.582\times10^{-22}$ MeV s; $c = 3.0\times10^{8}$ m/s;
$N_A = 6.022\times10^{23}$ mol$^{-1}$; 1 b $= 10^{-24}$ cm$^2 = 100$ fm$^2$, 1 pb $= 10^{-36}$ cm$^2$.

**Masas.** $m_{\pi^0} = 134.98$ MeV, $m_p = 938.27$ MeV, $m_\tau = 1776.9$ MeV, $m_Z = 91.19$ GeV.